Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -339,6 +339,14 @@ void WebsiteDataStore::resolveDirectoriesIfNecessary()
         m_resolvedConfiguration->setCookieStorageFile(FileSystem::pathByAppendingComponent(resolvedCookieDirectory, FileSystem::pathFileName(m_configuration->cookieStorageFile())));
     }
 
+    // Do not back up cache type data.
```

Comment:
You can write out your comment in code like this:

```
Vector<String> allCacheDirectories = {
    resolvedApplicationCacheDirectory(),
    resolvedMediaCacheDirectory(),
    resolvedNetworkCacheDirectory(),
#if ENABLE(ARKIT_INLINE_PREVIEW)
    resolvedModelElementCacheDirectory(),
#endif
};

for (const auto& directory : allCacheDirectories) {
    FileSystem::setExcludedFromBackup(directory, true);
};
```

Future diffs that add or remove cache directories get simpler this way too.

Reply:
Do we really need a Vector though, wouldn't an array suffice?
```
std::array allCacheDirectories = {
    resolvedApplicationCacheDirectory(),
    resolvedMediaCacheDirectory(),
    resolvedNetworkCacheDirectory(),
#if ENABLE(ARKIT_INLINE_PREVIEW)
    resolvedModelElementCacheDirectory(),
#endif
};
```

Reply:
That's good too! I think std::array might still require that you specify the count as a template parameter.

Reply:
> I think std::array might still require that you specify the count as a template parameter.
Not with C++17 AFAIK.

Reply:
https://en.cppreference.com/w/cpp/container/array/deduction_guides